### PR TITLE
NEXT-20706 - Add the missing keys to the shopware.yaml configuration

### PR DESCRIPTION
**Pull request by King-of-Babylon**
For the downloadable products feature we added two new configuration keys to the shopware.yaml.
In this PR we are updating the documentation with this new keys.

### CODE REVIEW

**Review by keulinho — APPROVED** [review]

**Review by keulinho — CHANGES_REQUESTED** [review]

File: guides/hosting/infrastructure/filesystem.md [hunk 1]
```diff
@@ -49,6 +49,21 @@ shopware:
...
+```yaml
+shopware:
+  filesystem:
+    public:
+      # The Adapter Configuration
+    private:
+      # The Adapter Configuration
+    allowed_extensions: # Array with allowed file extensions for public filesystem
+    private_allowed_extensions: # Array with allowed file extensions for private filesystem
+    private_local_download_strategy: # Name of the download strategy: php, x-sendfile or x-accel
```
[the remark this answers is not in the document]
Comment (King-of-Babylon) [reply guides/hosting/infrastructure/filesystem.md:64]: Done. See line >=  67

File: guides/hosting/infrastructure/filesystem.md [hunk 2]
```diff
@@ -49,6 +49,26 @@ shopware:
...
+```
+
+Following download strategies are valid:
```
Comment (keulinho) [inline guides/hosting/infrastructure/filesystem.md:67]: can you mention when this download strategy will be used? e.g. that it is only relevant for download products

Comment (King-of-Babylon) [reply guides/hosting/infrastructure/filesystem.md:67]: See in line 53, is this enough? :thinking:

File: guides/hosting/infrastructure/filesystem.md [hunk 3]
```diff
@@ -49,6 +49,26 @@ shopware:
       # The Adapter Configuration
 ```
 
+If you want to regulate the uploaded file types, then you could add the keys `allowed_extensions`for the public filesystem or `private_local_download_strategy` for the private filesystem.
+With the `private_local_download_strategy` key you could choose the download strategy for private files (e.g., the downloadable products):
```
Comment (keulinho) [inline guides/hosting/infrastructure/filesystem.md:53]:
```suggestion
With the `private_local_download_strategy` key you could choose the download strategy for private files (e.g. the downloadable products):
```

**Review by keulinho — APPROVED** [review]

**Pull request merged**